control/trajectory_follower/lane_departure_checker/include/lane_departure_checker/lane_departure_checker_node.hpp

```diff
@@ -39,6 +41,12 @@
 
 namespace lane_departure_checker
 {
+using autoware_auto_mapping_msgs::msg::HADMapBin;
+using autoware_auto_planning_msgs::msg::HADMapRoute;
+using autoware_auto_planning_msgs::msg::Trajectory;
+using autoware_auto_planning_msgs::msg::TrajectoryPoint;
+using TrajectoryPointArray = std::vector<autoware_auto_planning_msgs::msg::TrajectoryPoint>;
```

Comment:
Why did you write this here and in `lane_departure_checker.hpp`?
I guess `lane_departure_checker_node.hpp` includes `lane_departure_checker.hpp` and no need to write here again.

Reply:
I removed unnecessary alias
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/499/commits/4c7a50050fa8e70b19f2b11a8d486ae807df344b